pyControl4/director.py

```diff
@@ -50,15 +177,22 @@ async def sendGetRequest(self, uri):
                 async with self.session.get(
                     self.base_url + uri, headers=self.headers
                 ) as resp:
-                    await checkResponseForError(await resp.text())
+                    try:
+                        await checkResponseForError(await resp.text())
+                    except KeyError as err:
```

Comment:
Why was this `KeyError` except added? In what cases does a `KeyError` occur when running `checkResponseForError()`? 

I would prefer to add logic to handle the error properly in the `checkResponseForError()` function instead of just throwing it out to the logger.

Reply:
I hit this while implementing another lookup method for categories. We can strip it out for now if you would like. If was just a safe catch if there were a key error.